src/main/java/com/uploadcare/api/Image.java

```diff
@@ -0,0 +1,118 @@
+package com.uploadcare.api;
+
+import com.uploadcare.data.FileData;
+import com.uploadcare.data.ImageInfo;
+
+import java.util.Date;
+
+/**
+ * Created by Egor Petushkov
+ * on 09.06.17.
+ */
+public class Image extends File {
+    public static final int UNKNOW_DPI = -1;
```

Comment:
I wanted to use it when dpi isn't supplied, but now I think I should use null instead.